packages/components/src/internal/url/URLResolver.ts

```diff
@@ -539,6 +539,20 @@ export const STORAGE_BOX_MAPPER = new ActionMapper('query', 'detailsQueryRow', r
     return undefined;
 });
 
+// plate-isPlated.api?sampleId=100
+// map to /rd/samples/100/plates
+export const IS_PLATED_MAPPER = new ActionMapper('plate', 'IsPlated', row => {
+    const url = row.get('url');
+    if (url) {
+        const params = ActionURL.getParameters(url);
+        const sampleId = params.sampleId;
```

Comment:
Also, this could should be slightly more defensive and check for the sampleId being present as `AppURL.create()` does not accept undefined/null.

```tsx
const params = ActionURL.getParameters(url);
const sampleId = params.sampleId;

if (sampleId) {
    return AppURL.create('rd', 'samples', sampleId, 'plates');
}
```